src/main/webapp/staticFeeds/sample_notifications.json

```diff
@@ -24,5 +24,19 @@
         "actionAlt" : "Activate Services",
         "dismissable" : true
       }
-    ]
+    ],
+  "dismissedNotifications" : [
```

Comment:
Pros/cons on a separate `dismissedNotifications` array vs a single array with items having a `dismissedDate` of today or earlier being automatically dismissed?  The latter approach moves parsing cost to run-time (rather than deploy-time pre-processing of this data into the different arrays), but it picks up the feature of being able to schedule out auto-dismiss into the future at the time the notification is published, no different `JSON` to publish to switch the notification state on the intended auto-dimiss-day.